### PR TITLE
Typo

### PR DESCRIPTION
**Pull request by jubalh**
### What does this PR do?

Fix typos. Trying to make the documentation more uniform, using the same spelling and making it generally a bit more readable.

### CODE REVIEW

File: susetest/python/susetest_api/system.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 import re
 
 
-# Thx to Aurelian Aptel, Samba Dev.
+# Thx to Aurelien Aptel, Samba Dev.
```
Comment (MalloZup) [inline susetest/python/susetest_api/system.py:12]: LOL

File: doc/first_steps.md [hunk 2]
```diff
@@ -390,36 +390,34 @@ def replace_string(node, replacements, _file, _max_replace=0):
...
+If you want to use the susetest_api, you need to import the specific modules, and function you want to use:
 
 ```
 from susetest_api.log import systemd_check
 
 systemd_check(sut)
 ```
 
-or  an example with more functions
+Aan example with more functions:
```
Comment (MalloZup) [inline doc/first_steps.md]: here a typo "An"

**Comment by MalloZup** [comment]
Hi Micheal, yop thx look nice ! say to me when you think you are done with the PR

File: doc/first_steps.md (hunk 2 above)
Comment (jubalh) [inline doc/first_steps.md]: Thanks!

**Comment by jubalh** [comment]
Hey Dario :)
Yeah I think I am finished. Just one thing:
I am not sure what you want to say with:

> This feature is under development (it works fine, but some feature, and testing should be done, before it will be documented). `Compare`, will call machinery compare command, with the actual system and a des    ↪ cription of a systemd stored.

**Comment by MalloZup** [comment]
i will remove the doc about the compare feature, because is not already stable implemented

**Comment by jubalh** [comment]
Okay I do it (am on the PC anyways)

**Comment by jubalh** [comment]
Can be merged now :)

**Pull request merged**